pkgs/tools/package-management/akku/overrides.nix

```diff
@@ -0,0 +1,135 @@
+{ stdenv, lib, akku, curl, git, substituteAll }:
+let
+  joinOverrides =
+    overrides: pkg: old:
+    lib.attrsets.mergeAttrsList (map (o: o pkg old) overrides);
+  addToBuildInputs =
+    extras: pkg: old:
+    { propagatedBuildInputs = old.propagatedBuildInputs ++ extras; };
+  broken = lib.addMetaAttrs { broken = true; };
+  skipTests = pkg: old: { doCheck = false; };
+  # debugging
+  showLibs = pkg: old: { preCheck = "echo $CHEZSCHEMELIBDIRS"; };
+  runTests = pkgs: old: { doCheck = true; };
+  brokenOnAarch64Darwin = pkgs: old: { broken = stdenv.isDarwin && stdenv.isAarch64; };
```

Comment:
I guess `broken` here should be `meta.broken` instead?

Reply:
In that case we should rather use `lib.addMetaAttrs { broken = stdenv.isDarwin && stdenv.isAarch64; }`